.github/workflows/bundle-size/package-lock.json

```diff
@@ -1,1536 +0,0 @@
-{
```

Comment:
Don't we need this lockfile to pin build tools versions (Vite)?

Also, since https://github.com/cloudscape-design/components/pull/2031, Cloudscape packages are gone from this package-lock file

Reply:
With the lockfile, the build unfortunately fails: https://github.com/cloudscape-design/components/actions/runs/8171937882/job/22341231084#step:6:16
According to the Rollup log output, this appears to be a bug in NPM, where the lockfile apparently needs to be created on the same platform and architecture as the consumer of the lockfile

Reply:
Coincidentally, I am currently fixing the same build error in codeview: https://github.com/cloudscape-design/code-view/pull/22

It was easily fixable by doing a clean install.

Having lockfile is a must here. You do not want to block the entire team PR process, when an expected change in Vite or Typesctipt happens and bundle size step breaks and blocks PRs from merging

Reply:
Ah, indeed - I had to regenerate the lockfile locally two times but now it works 😄